src/WinFormsUI/Docking/DockPane.cs

```diff
@@ -955,7 +952,6 @@ protected override void OnLayout(LayoutEventArgs levent)
             base.OnLayout(levent);
         }
 
-        [SecurityPermission(SecurityAction.LinkDemand, Flags = SecurityPermissionFlag.UnmanagedCode)]

```

Comment:
Whats the reasoning behind removing this warning? It still unmanaged code and might lead to problems?

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.security.permissions.securitypermission?view=windowsdesktop-9.0&viewFallbackFrom=net-9.0&form=MG0AV3